docs/data/update-oasis.md

```diff
@@ -35,34 +35,32 @@ The next step is to generate a proxy and log into `oasis-login.opensciencegrid.o
 user@host $ voms-proxy-init -voms %RED%VO%ENDCOLOR%
 user@host $ gsissh -o GSSAPIDelegateCredentials=yes oasis-login.opensciencegrid.org
 ```
-
+<br/>
```

Comment:
What's the break here for?

If it's to improve the visual lines between formatted text and non-formatted, that should be fixed in the CSS.

I really don't want these docs to grow a lot of HTML like the twiki did, especially if it's to tweak formatting that could be done at the presentation layer.

Reply:
Yes that's what it is for.  I have used that on the other pages I own as well.  If it can be fixed in CSS, great, but I don't know how to do it.

Reply:
My understanding is that we are going to deal with CSS fixes later. And I would volunteer myself for that task – it’s been a while, but I once taught a small course on CSS and became fairly handy with it.

Reply:
Alright - given that input, @DrDaveD can you do a subsequent commit to remove the breaks?  Then, I'll be happy to merge.

(If you don't have time, just let me know and I'll make the commit for you)

Reply:
I don't want it to look ugly.  Why not leave in the br statements until the CSS is fixed?  When will it be fixed?

Reply:
Simple reasoning: it looks consistent with the rest of the docs.  When we change the CSS defaults, this doc will look ugly until this change is backed out.

Also: I don't want HTML in our docs except for places where we simply have no choice.  This doesn't get past that threshold.

I'll happily make the changes for you, but I don't want to merge it with all the new `<br/>` instances.